packages/client-core/src/admin/services/SceneService.ts

```diff
@@ -40,7 +41,7 @@ export const AdminSceneState = defineState({
     fetched: false,
     updateNeeded: true,
     lastFetched: Date.now(),
-    singleScene: { scene: {} } as SceneDataType
+    singleScene: { scene: {} } as Omit<SceneDataType, 'scene'> & { scene: SceneJsonType }
```

Comment:
the type gymnastics here are a bit funky. Is typebox not getting the right type from the schema?

Reply:
No, the type was dependent on the ecs package, which common can not have. This stuff will be replaced with gltf soon, so it's more or less temporary.

Reply:
oh scrolled down. Sorry. Was working off local from when I clicked in github. I guess it changes the question to. You can't pass SceneJsonType to typebox?

Reply:
We might be able to